process/wg-lifecycle-documents/ai_ml_incubating_stage.md

```diff
@@ -0,0 +1,65 @@
+## AI/ML Working Group incubation application
+
+### List WG Chairs
+
+The WG must have a minimum of 1 Chair
+
+* Mihai Maruseac, Google, mihaimaruseac
+* Jautau "Jay" White, Microsoft, camaleon2016
+
+### Working Group (WG) has met all Sandbox requirement
+
+* Applying directly to Incubating, but we have met Sandbox requirements, as
+  listed below
+  * Proposal of scope has been reviewed by TAC: [#175](https://github.com/ossf/tac/issues/175) (and [#188](https://github.com/ossf/tac/issues/188) to ensure limited overlap)
+  * At least 3 interested individuals, from 2 different organizations supporting the proposal. See list of regular contributors in the next section
+  * 1 TAC sponsor. Originally Dan Appelquist, now Jay White
+
+### List of regular contributors
+
+The WG must have a minimum of 5 contributors from at least 3 different
+organizations attending regularly.
+
+* David A. Wheeler, LF
+* David Edelsohn, IBM / AI Alliance
+* Emily Fox, Red Hat
+* Jautau "Jay" White, Microsoft
+* Karen Bennet, ISO
+* Marcela Melara, Intel
+* Mihai Maruseac, Google
+* Newt Tan, University of Glasgow
+* Sarah Evans, Dell Technologies
+* Yotam Perkal, Rezilion
+
+and many others
+
+### Mission of the Working Group
+
+The WG must have a charter or mission statement for review by TAC
+
+* https://github.com/ossf/ai-ml-security/blob/main/CHARTER.md
+* https://github.com/ossf/ai-ml-security/blob/main/mvsr.md
+
+### Governance
+
+WG must have documented, initial group governance.
+
+* https://github.com/ossf/ai-ml-security/blob/main/README.md#wg-leadership
+
+WG must have met publicly at least 5 times in the last quarter since becoming
+Sandbox
+
+* 2024: https://docs.google.com/document/d/1YNP-XJ9jpTjM6ekKOBgHH8-avAws2DVKeCpn858siiQ/edit
+* 2023: https://docs.google.com/document/d/1hEJdr7yXv5tfYpUfn_e7DAUTyvoU6mY0b9eH6EXAHaI/edit
+
+WG must have defined Contributor Guide
+
+* https://github.com/ossf/ai-ml-security/blob/main/README.md#how-to-participate
```

Comment:
Just a really minor nit: The Slack channel linked in the README still points to the old name that used underscores. You may want to update that so it points to the updated `#wg-ai-ml-security` channel.

Reply:
Thank you for catching that! I updated the repo's README.